backend/xivdpscalc/util/types.py

```diff
@@ -0,0 +1,6 @@
+"""Contains custom types for easier readability"""
+from xivdpscalc.pps import Pps
+
+# When converting a string into tuple with job, mainstat, and potencycalc
+MainstatString = str
```

Comment:
I'm unsure why this type is needed. It seems like a thin wrapper around a string type

Reply:
removed